objects/obj_shop/Draw_0.gml

```diff
@@ -10,8 +11,8 @@ draw_set_color(c_gray);
 draw_set_halign(fa_left);
 
 
-var te;te="";if (target_comp=1) then te="1st";if (target_comp=2) then te="2nd";
-if (target_comp=3) then te="3rd";if (target_comp>3) then te=string(target_comp)+"th";
+var te="";
+if (target_comp<=10) then te=romanNumerals[target_comp-1];
```

Comment:
could this be ever >10? and if so, what should be done here?

Reply:
So far I dont think so, but i cant find where target_comp is defined. If its more than 10 probably its HQ or one of the others non company organizations in the panel. Need to see more of the logic. Will mark as a TODO for now and we could add a new PR for how the company is defined?

Reply:
ok, mark as TODO